### PR TITLE
Bug fixed: Will always set the Short Display times + Refact

### PR DESCRIPTION
**Pull request by ivandrofly**

### CODE REVIEW

File: src/Forms/ApplyDurationLimits.cs [hunk 1]
```diff
@@ -142,14 +141,13 @@ private void FixShortDisplayTimes()
                 if (displayTime < minDisplayTime)
                 {
                     Paragraph next = _working.GetParagraphOrDefault(i + 1);
-                    if (next == null || (p.StartTime.TotalMilliseconds + minDisplayTime < next.StartTime.TotalMilliseconds))
+                    if (next == null || (p.StartTime.TotalMilliseconds + minDisplayTime < next.StartTime.TotalMilliseconds) && AllowFix(p))
```
Comment (ivandrofly) [inline src/Forms/ApplyDurationLimits.cs]: Bug found here

File: src/Forms/ApplyDurationLimits.cs [hunk 2]
```diff
@@ -128,7 +128,6 @@ private void FixShortDisplayTimes()
         {
             Subtitle unfixables = new Subtitle();
             string fixAction = Configuration.Settings.Language.FixCommonErrors.FixShortDisplayTime;
-            int noOfShortDisplayTimes = 0;
```
Comment (ivandrofly) [inline src/Forms/ApplyDurationLimits.cs]: Not used

File: src/Forms/ApplyDurationLimits.cs [hunk 3]
```diff
@@ -165,23 +163,18 @@ private void FixShortDisplayTimes()
...
-            int noOfLongDisplayTimes = 0;
             for (int i = 0; i < _working.Paragraphs.Count; i++)
             {
                 Paragraph p = _working.Paragraphs[i];
                 double displayTime = p.Duration.TotalMilliseconds;
                 double maxDisplayTime = (double)numericUpDownDurationMax.Value;
-                if (displayTime > maxDisplayTime)
+                if (displayTime > maxDisplayTime && AllowFix(p))
                 {
-                    if (AllowFix(p))
```
Comment (ivandrofly) [inline src/Forms/ApplyDurationLimits.cs]: Moved above

File: src/Forms/ApplyDurationLimits.cs (hunk 1 above)
Comment (niksedk) [inline src/Forms/ApplyDurationLimits.cs]: :)

**Pull request merged**